dictionaries.py

```diff
@@ -0,0 +1,125 @@
+
+class Dictionaries:
+    def __init__(self):
+        pass
```

Comment:
Alternatively, you could create all the dictionaries within the constructor function and store them as attributes on self. If you do this, you don't need the methods any more.

```suggestion
    def __init__(self):
        self.animals = { ... }
        self.fruits = { ... }
        # etc.
```

So then elsewhere in the code, you would access as:

```
d = Dictionaries()
d.animals  # this gives you the dictionary
```

Reply:
Thank you for the this suggestion. It makes the code a lot more cleaner and uncomplicates the code in the `play module` for picking a category